app/src/main/res/layout/selection_interaction_item.xml

```diff
@@ -46,7 +46,7 @@
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="14sp"
       android:textStyle="italic"
-      android:visibility="@{viewModel.getSelectionItemInputType() == SelectionItemInputType.RADIO_BUTTONS ? View.GONE : View. VISIBLE}" />
+      android:visibility="@{viewModel.getSelectionItemInputType() == SelectionItemInputType.RADIO_BUTTONS || viewModel.selectedItemsCount > 0 ? View.GONE : View. VISIBLE}" />
```

Comment:
Given the OR here, do we already have tests verifying that the text is always showing for radio buttons for both unselected & selected states to contrast with the checkbox behavior?

Reply:
@BenHenning  No, I will add them.

Reply:
@BenHenning For the radio buttons the text will always be hidden. I am not sure we need tests for this.

Reply:
@BenHenning I have added test case for the radio buttons.

Reply:
@BenHenning state 2 is Fraction input option. State 3 and state 4 are  multiple choice input. I have already added the test case for state 4.

```
@Test
 fun testStateFragment_interactions_radioItemSelection_selectionTextIsNotDisplayed() {
     launchForExploration(TEST_EXPLORATION_ID_2, shouldSavePartialProgress = false).use {
      startPlayingExploration()
      playThroughPrototypeState1()
      playThroughPrototypeState2()
      playThroughPrototypeState3()

      // Verify that the user is now on the fourth state.
      verifyViewTypeIsPresent(SELECTION_INTERACTION)
      onView(withId(R.id.selection_interaction_textview)).check(matches(not(isDisplayed())))
    }
  }

```
//Added these additional test cases
```
testStateFragment_interactions_singleRadioItemSelected_selectionTextIsNotDisplayed
testStateFragment_interactions_multipleRadioItemSelected_selectionTextIsNotDisplayed
testStateFragment_interactions_correctRadioItemSelected_selectionTextIsNotDisplayed
```